### PR TITLE
MINOR:fix small label issue

### PR DESCRIPTION
**Pull request by TaiJuWu**
See https://github.com/apache/kafka/pull/17260/files#r1789229563

The original implementation will cause `./.github/scripts/label_small.sh: line 22: printf: warning: s': invalid time format specification`

Replace `gh pr diff`  with `gh pr view`.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: .github/scripts/label_small.sh [hunk 1]
```diff
@@ -17,10 +17,10 @@
 LABEL_NAME=small
 MAX_SIZE=100
 
-pr_diff=$(gh pr diff $PR_NUM -R $GITHUB_REPOSITORY)
+pr_diff=$(gh pr view $PR_NUM -R $GITHUB_REPOSITORY --json additions,deletions)
 
-insertions=$(printf "$pr_diff" | grep '^+' | wc -l)
-deletions=$(printf "$pr_diff" | grep '^-' | wc -l)
+insertions=$(echo "$pr_diff" | jq -r '.additions')
```
Comment (mumrah) [inline .github/scripts/label_small.sh]: maybe we should use the same terminology here and call this "additions"

**Review by mumrah — APPROVED** [review]
LGTM with one minor suggestion

File: .github/scripts/label_small.sh (hunk 1 above)
Comment (TaiJuWu) [reply .github/scripts/label_small.sh]: Done. Thanks for review.

**Comment by chia7712** [comment]
@TaiJuWu could you please temporarily merge #17338 into this PR to confirm that the bug is fixed?

**Comment by TaiJuWu** [comment]
> @TaiJuWu could you please temporarily merge #17338 into this PR to confirm that the bug is fixed?

Done. It has been fixed.

**Review by chia7712 — APPROVED** [review]

**Comment by chia7712** [comment]
> Done. It has been fixed.

Did you notice that the small tag was removed after merging #17338?

**Comment by chia7712** [comment]
> Did you notice that the small tag was removed after merging https://github.com/apache/kafka/pull/17338?

please ignore this comment. The label action runs off trunk. I will merge it and verify on #17338

**Pull request merged**